src/modules/workspaces/workspaces.usecase.spec.ts

```diff
@@ -949,4 +951,214 @@ describe('WorkspacesUsecases', () => {
+        member: user,
+      });
+
+      const workspaceUsers = jest
+        .spyOn(workspaceRepository, 'findWorkspaceUsers')
+        .mockResolvedValue(Promise.resolve([workspaceUser]));
+
+      const teamsAndMembers = jest
+        .spyOn(teamRepository, 'getTeamsAndMembersCountByWorkspace')
+        .mockResolvedValue([]);
+
+      const mockWorkspaceUsers: WorkspaceUserMemberDto[] = [
+        {
+          ...workspaceUser.toJSON(),
+          isOwner: false,
+          isManager: false,
+          freeSpace: workspaceUser.getFreeSpace(),
+          usedSpace: workspaceUser.getUsedSpace(),
+        },
+      ];
+
+      const fnGetMembers = service.getWorkspaceMembers(workspace.id, owner);
+
+      await expect(fnGetMembers).resolves.toStrictEqual(mockWorkspaceUsers);
+      await expect(fnGetMembers).resolves.toMatchObject([{ isManager: false }]);
+
+      expect(workspaceUsers).toHaveBeenCalled();
+      expect(teamsAndMembers).toHaveBeenCalled();
+    });
+
+    it('When a user is registered as Manager within a team then the "isManager" field will be True', async () => {
+      const user1 = newUser();
+      const user2 = newUser();
+
+      const workspaceUser1 = newWorkspaceUser({
+        workspaceId: workspace.id,
+        memberId: user1.uuid,
+        member: user1,
+      });
+      const workspaceUser2 = newWorkspaceUser({
+        workspaceId: workspace.id,
+        memberId: user2.uuid,
+        member: user2,
+      });
+
+      const workspaceTeam = newWorkspaceTeam({
+        workspaceId: workspace.id,
+        manager: user1,
+      });
+
+      newWorkspaceTeamUser({
+        teamId: workspaceTeam.id,
+        team: workspaceTeam,
+        memberId: user1.uuid,
+      });
+      newWorkspaceTeamUser({
+        teamId: workspaceTeam.id,
+        team: workspaceTeam,
+        memberId: user2.uuid,
+      });
+
+      const mockWorkspaceUser = Promise.resolve([
+        workspaceUser1,
+        workspaceUser2,
+      ]);
+      const workspaceUsers = jest
+        .spyOn(workspaceRepository, 'findWorkspaceUsers')
+        .mockResolvedValue(mockWorkspaceUser);
+
+      const mockTeamsAndMembers = Promise.resolve([
+        { team: workspaceTeam, membersCount: 2 },
+      ]);
+      const teamsAndMembers = jest
+        .spyOn(teamRepository, 'getTeamsAndMembersCountByWorkspace')
+        .mockResolvedValue(mockTeamsAndMembers);
+
+      const workspaceMembers: WorkspaceUserMemberDto[] = [
+        {
+          ...workspaceUser1.toJSON(),
+          isOwner: false,
+          isManager: true,
+          freeSpace: workspaceUser1.getFreeSpace(),
+          usedSpace: workspaceUser1.getUsedSpace(),
+        },
+        {
+          ...workspaceUser2.toJSON(),
+          isOwner: false,
+          isManager: false,
+          freeSpace: workspaceUser2.getFreeSpace(),
+          usedSpace: workspaceUser2.getUsedSpace(),
+        },
+      ];
+
+      const getMembers = service.getWorkspaceMembers(workspace.id, owner);
```

Comment:
You can use

```
const getMembers = await service.getWorkspaceMembers(workspace.id, owner);
await expect(getMembers).toStrictEqual(workspaceMembers);
      await expect(getMembers).toMatchObject([
        { isManager: true },
        { isManager: false },
      ]);
```

Yo do not need to use resolves every time, async await are the way to go if you want something cleaner.